android/src/main/java/so/onekey/app/wallet/lite/onekeyLite/nfc/Connection.kt

```
@@ -42,8 +48,11 @@ class Connection(val isoDep: IsoDep, private val mCommandGenerator: CommandGener
                 val sw2 = response[response.size - 1]
 
                 return SendResponse(byteArr2HexStr(response), sw1, sw2, byteArr2HexStr(resp))
+            } catch (e: TagLostException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)